### ISSUE TITLE
Enhancement : limited correction available

### ISSUE THREAD

**Issue by stmolivier** [iCAPLyon1/DropZoneBundle#121]
Maybe i didn't understand the use or the purpose of the evaluation Resssource : 
When a teacher adds a correction himself, all he can do is give a mark (x/20), which is i think a little dry  ! 
Maybe at least a rich text to explain some point justifying his notation could be useful or even better a rich text  for each ressource dropped.
At least (!) when he choose peer correction, he can enforce a comment from the corrector.

**Comment by bradInside** [comment]
you should have an option "Allow comments" in dropzone options that allow corrector to put some text with the grade , and you can force corrector to give comments. Maybe, i'm wrong, i 'm no more on the project since few month but in my mind, its possible ;)

**Comment by stmolivier** [comment]
yes that' is what i said in my last sentence : when peers correct each other the teacher can check a ccheckbox to make a comment mandantory. 
But if the teacher is the corrector himself (he chose the option to correct himself, and not correct by peers), when he is on a drop i only see he can give a grade (xx/20), no other comment zone.

Well, only if the ressource dropped is text, can he open it and modify it (which isn't great for future contestation...) !
